include/inviwo/core/properties/optionproperty.h

```diff
@@ -6,7 +6,7 @@
  * All rights reserved.
  *
  * Redistribution and use in source and binary forms, with or without
- * modification, are permitted provided that the following conditions are met:
+ * modification, are permitted provided that thvoide following conditions are met:
```

Comment:
?

Reply:
oh yeah.. I guess that was my cursor flying all over the place :smile: 
fixed it